src/components/About.js

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import { social } from "./socialMedia"
+import styles from "../css/soc-links.module.scss"
+import { Link } from "gatsby"
+
+export default function About() {
+  const links = [...social]
+  return (
+    <>
+      <section className="about-grid">
+        <div className="about-text">
+          <h3>
+            <span className="anim-type">Hi, I`m Liuba</span>
+          </h3>
+          <h1>Web Developer</h1>
+          <Link to="/contact">
+            <button>Contact me</button>
+          </Link>
+          <h3>
+            {" "}
+            I will code your dreams into{" "}
+            <span className="anim-flash">reality</span>
+          </h3>
```

Comment:
Remember what I said about headings? The should be meaningful and follow a predictable order. Starting from h1, then h2, h3, etc... Imagine the page like a book, without fancy CSS styles, just plain "raw markdown styles". Does it make sense for this to be an h3?

Reply:
Another tip. What's the ideal title of this page? That should be the h1, nothing else, and the first heading. Even if it doesn't make sense visually, you can hide it using `.visually-hidden` ;)